src/main/java/service/AgendaService.java

```diff
@@ -4,17 +4,28 @@
 
 import model.Course;
 import model.Student;
+import repository.StudentRepository;
+import repository.TransactionManager;
 
 public class AgendaService {
 
+	private TransactionManager transactionManager;
+
+	public AgendaService(TransactionManager transactionManager) {
+		this.transactionManager = transactionManager;
+	}
+
 	public List<Student> getAllStudents() {
-		// TODO Auto-generated method stub
-		return null;
+		return transactionManager.studentTransaction(StudentRepository::findAll);
 	}
 
 	public Boolean findStudent(Student student) {
-		// TODO Auto-generated method stub
-		return null;
+		return transactionManager.studentTransaction(studentRepository -> {
+			if (studentRepository.findById(student.getId()) != null)
```

Comment:
We have to find a way to make it compliant. I thought about the ternary expression but not knowing how to implement it in a lambda I used the simplest way